### ISSUE TITLE
Show reserved DOI for all versions on new upload

### ISSUE THREAD

**Issue by santisoler** [zenodo/zenodo#1951]
When creating a new upload we can reserve a DOI for the first version of the files we are about to upload. This allows to include the reserved DOI on the files to upload, which is of great help when we want to add instructions on how to cite our work within the uploaded files.

For projects that will surely have several versions (e.g. software), it's better to point citations to the _DOI for all versions_ instead of the DOI of any specific version.

![imagen](https://user-images.githubusercontent.com/11541317/75552347-de153900-5a14-11ea-9f57-b85ff734ed11.png)

Would be nice to see the reserved _DOI for all versions_ while drafting the new upload along with the reserved DOI for the first version we are about to release.

Is it possible?
Thanks!

cc @leouieda

**Comment by frankois** [comment]
Unfortunately, this is not yet possible. It is something we want to allow in the future, but until then you can use this small workaround:

Being logged-in, go to the following page: `https://zenodo.org/api/deposit/depositions/<id>` with `<id>` being the last part of your DOI. 
> In your case, your DOI for version v0.2.0 being `10.5281/zenodo.3627166` , this would be `3627166`.

You will then be able to access the `conceptdoi` field which is the *DOI for all version* (as shown in the picture below)

https://zenodo.org/api/deposit/depositions/3627166

![Screen Shot 2020-07-14 at 4 41 22 PM](https://user-images.githubusercontent.com/4917492/87439301-e4d1d080-c5f0-11ea-99f4-75627dd995b2.png)

**Comment by santisoler** [comment]
Thanks @frankois for the response.

If I understood correctly, this workaround you suggest can only be carried out when the upload has already been published, right?
Or can I access the `conceptdoi` even if the upload is yet a draft?

**Comment by frankois** [comment]
@santisoler No, you can actually access the `conceptdoi` even if the upload is a draft.

**Comment by santisoler** [comment]
Great news then! :rocket:
Thanks again for the hack!
I'm sure it will be helpful until the `conceptid` is shown while dafting.
Hope you can implement it in the future!
